### PR TITLE
craeted find_prompt_special_case in telnet login to sleep between reads

### PR DESCRIPTION
**Pull request by shreyashah**
several cleanup stage failures seen intermittent when connecting to router via console. 
the issue is not 100% reproducible. But it is seen intermittently.

Issue logs: http://ott-idt-jenk1:8080/job/test_cafyap_cleanup/97715/consoleFull - POD8
http://ott-idt-jenk1:8080/job/test_cafyap_cleanup/97725/consoleFull - POD10

Logs with this fix: /ws/shreyash-sjc/temp/bak_netmiko.log

Logs without this fix: /ws/shreyash-sjc/temp/netmiko.log

### CODE REVIEW

File: netmiko/cisco_base_connection.py [hunk 1]
```diff
@@ -85,6 +85,63 @@ def serial_login(
...
+            except ReadTimeout:
+                pass
+        else:
+            # Initial read
+            time.sleep(sleep_time)
+            prompt = self.read_channel().strip()
+
+            count = 0
+            while count <= 12 and not prompt:
+                if not prompt:
```
Comment (akshansh237) [inline netmiko/cisco_base_connection.py]: not needed

File: netmiko/cisco_base_connection.py [hunk 2]
```diff
@@ -85,6 +85,63 @@ def serial_login(
...
+                    self.write_channel(self.RETURN)
+                    t = 0
+                    while t <= 60:
```
Comment (akshansh237) [inline netmiko/cisco_base_connection.py]: while t<=60 and not prompt

File: netmiko/cisco_base_connection.py [hunk 3]
```diff
@@ -85,6 +85,63 @@ def serial_login(
...
+                            time.sleep((delay_factor * 0.1) + 5)
+                            prompt = self.read_channel()
+                        cxr_pattern = "last switch-over"
+                        if cxr_pattern in prompt.lower():
+                            time.sleep((delay_factor * 0.1) + 3)
+                            prompt = self.read_channel()
+                        if prompt:
+                            break
+                count += 1
+
```
Comment (akshansh237) [inline netmiko/cisco_base_connection.py:134]: remove this

File: netmiko/cisco_base_connection.py [hunk 4]
```diff
@@ -85,6 +85,63 @@ def serial_login(
...
+                        prompt = self.read_channel().strip()
+                        autocommand_pattern = "executing autocommand"
+                        if autocommand_pattern in prompt.lower():
+                            time.sleep((delay_factor * 0.1) + 5)
+                            prompt = self.read_channel()
+                        cxr_pattern = "last switch-over"
+                        if cxr_pattern in prompt.lower():
+                            time.sleep((delay_factor * 0.1) + 3)
+                            prompt = self.read_channel()
+                        if prompt:
```
Comment (akshansh237) [inline netmiko/cisco_base_connection.py]: remove this

File: netmiko/cisco_base_connection.py [hunk 5]
```diff
@@ -122,7 +179,8 @@ def telnet_login(
                     if output == '':
                         time.sleep(2 * delay_factor)
                         self.log.debug("output is empty, doing find_prompt()")
-                        output = self.find_prompt()
+                        #output = self.find_prompt()
```
Comment (akshansh237) [inline netmiko/cisco_base_connection.py:179]: remove commented code

**Review by akshansh237 — COMMENTED** [review]
minor comments

**Comment by shreyashah** [comment]
fretta logs:
 /ws/shreyash-sjc/temp_pytest_logs3/fretta_netmiko.log
 
 spitfire logs:
/ws/shreyash-sjc/temp_pytest_logs3/spitfire_netmiko.log

**Pull request merged**